meteor/server/api/rundownLayouts.ts

```diff
@@ -116,6 +141,26 @@ PickerGET.route('/shelfLayouts/download/:id', async (params, _req: IncomingMessa
 	res.end(content)
 })
 
+function validateAndGetRestoredLayoutBody(req: IncomingMessage): RundownLayoutBase {
```

Comment:
Since we only work on the `req.body`, perhaps the parameter could be reduced to taking only the body?

Reply:
I think it's better that picking the right property form the `IncomingMessage` is the responsibility of this function. Otherwise it would need to accept `string | object | undefined` which can break more easily in case of refactoring.